### DISCUSSION TITLE
Error getting F-Droid index file.

### DISCUSSION THREAD

**Discussion by ArigatoDevs** [bromite/bromite#1212] [General]
Hi Bromite Team.
I was trying to update Bromite through Fdroid and while updating the repositories this came-

Bromite official F-Droid repository:
Error getting F-Droid index file-No 
mirrors available
All other repos didn't create errors.

It's happening on my other devices too. Any way to fix it? 
Thank you.

**Comment by ArigatoDevs** [comment]
Does anyone has any idea why this error occured?

**Reply by csagan5** [reply]
Certificate renovations error; I have fixed it in past days.

**Reply by ArigatoDevs** [reply]
Ah I see. Thank you for the reply. Love Bromite.

**Reply by ArigatoDevs** [reply]
There is another small bug that I am seeing, Bromite apps don't show their logo now in F Droid or any other clients such as AuroraDroid and FoxyDroid. I have checked on other devices as well, it's not showing.

**Reply by csagan5** [reply]
It should be fixed now.

**Reply by ArigatoDevs** [reply]
> It should be fixed now.

You are the best. Many thanks!

**Comment by galvanized** [comment]
Having issues with the F-Droid repo with the same "no mirrors available" mentioned in the first post.

Using a browser to visit `https://fdroid.bromite.org/fdroid/repo/` gives a 403 Forbidden page.

F-Droid version 1.12.1.

**Reply by csagan5** [reply]
There are currently no issues with the F-Droid repository.

**Reply by galvanized** [reply]
I have since tried a different device and it worked, but still not on the original device. Obviously a different problem, sorry for the misleading report.
